images/armhf/python2-unicornhat.dev/README-FR.md

```diff
@@ -1,12 +1,12 @@
 Image Docker de développement pour le Unicorn sur Raspberry PI

```

Comment:
Thanks for your contribution! I was wondering if we could move the translation into a blog post or Gist? I'd rather not have other languages inside the repo because I won't be able to maintain them personally.